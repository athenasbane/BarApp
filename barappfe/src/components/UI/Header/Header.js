import React from 'react';
import { makeStyles } from '@material-ui/core';

import Menu from '../../Menu/Menu';

const useStyles = makeStyles({
  root: {
    width: '100%',
    position: 'fixed',
    left: 0,
    top: 0,
    display: 'flex',
    justifyContent: 'center',
    textAlign: 'center',
  },
});

const Header = () => {
  const classes = useStyles();
  return (
    <div className={classes.root}>
      <Menu />
    </div>
  );
};

export default Header;
