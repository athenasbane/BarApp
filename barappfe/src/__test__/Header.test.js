import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import { Header } from '../components/UI/Header/Header';

describe('Body Tests', () => {
    test('renders correctly', () => {
        const tree = renderer
            .create(<Header />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
}); 