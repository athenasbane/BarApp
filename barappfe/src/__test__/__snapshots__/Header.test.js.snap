// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body Tests renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <div
    className="makeStyles-root-2"
  >
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-button-5 MuiButton-textSecondary"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Menu
      </span>
    </button>
  </div>
</div>
`;
